### ISSUE TITLE
[BUG] Board options are not showing for custom (Sanguino) boards

### ISSUE THREAD

**Issue by jaxzin** [Renaud11232/OctoPrint-Marlin-Flasher#90]
**Is your feature request related to a problem? Please describe.**
My Melzi board needs to use the following fqbn value: Sanguino:avr:sanguino:cpu=atmega1284p
But I don't appear to have a way to append the cpu.

**Describe the solution you'd like**
The ability to enter the fqbn myself, or a GUI that supports appending the cpu value.

**Comment by Renaud11232** [comment]
Hello,

This should be working, normally all the options should be displayed. If they are not, there might be a bug somewhere.

Could you please provide the logs (`octoprint.log`) so see if there is any error there ?

I'm guessing the plugin is configured to use arduino-cli in your case. Can you tell what version of arduino-cli is installed ?

Thanks

**Comment by jaxzin** [comment]
I think this is a bit of user error. I first installed the Sanguino Core and was trying to Flash my device. That failed because I also needed the arduino:avr Core too. I didn't reload the interface at the time and while Sanguino appeared as an option under Board, the Processor field didn't appear. When I came back later the Processor field appeared under the Flash section and I was able to Flash my device successfully.
